bugbug/bugzilla.py

```diff
@@ -75,8 +76,16 @@ def get_bug_fields():
     return r.json()["fields"]
 
 
-def get_bugs():
-    return db.read(BUGS_DB)
+def get_bugs(include_invalid=False):
+    bugs = db.read(BUGS_DB)
+    for bug in bugs:
```

Comment:
You should be able to implement it in a more concise way, something like:
```Python
yield from (bug for bug in bugs if include_invalid or bug["product"] != "Invalid Bugs"
```
Note: we have to exclude all bugs from "Invalid Bugs", not only "Invalid Bugs::General".